tests/app/dao/test_provider_rates_dao.py

```diff
@@ -4,7 +4,7 @@
 from app.models import ProviderRates, ProviderDetails
 
 
-def test_create_provider_rates(notify_db, notify_db_session, mmg_provider):
+def test_create_provider_rates(mmg_provider):
```

Comment:
I'm not sure if this is correct (and it's my change).  Is the new provider rate still in the database after the test?